string id
int16 newResolution
---
